packages/emitter-framework/src/typescript/components/static-serializers.tsx

```diff
@@ -1,7 +1,8 @@
-import { code, refkey } from "@alloy-js/core";
+import { code } from "@alloy-js/core";
 import * as ts from "@alloy-js/typescript";
+import { efRefkey } from "../utils/refkey.js";
 
-export const DateRfc3339SerializerRefkey = refkey();
+export const DateRfc3339SerializerRefkey = efRefkey();
```

Comment:
since this is a zero-arg refkey (just need a unique value) these changes aren't strictly necessary but it's fine.